cmd/tesseract/posix/ci/docker-compose.yml

```diff
@@ -0,0 +1,59 @@
+services:
+  tesseract:
+    build:
+      context: ../../../../
+      dockerfile: cmd/tesseract/posix/ci/Dockerfile
+    container_name: tesseract
+    ports:
+      - "6962:6962"
+    command:
+      - "--http_endpoint=0.0.0.0:6962"
+      - "--storage_dir=/testlog"
+      - "--v=1"
+      - "--roots_pem_file=/testdata/test_root_ca_cert.pem"
+      - "--origin=example.com/test-ecdsa"
+      - "--private_key=/testlog/test-ecdsa-priv.pem"
+      - "--enable_publication_awaiter=false"
+    volumes:
+      - ../../../../internal/hammer/testdata:/testdata:ro
```

Comment:
Is this required?

Reply:
Yeah, we use it for the roots pem: `"--roots_pem_file=/testdata/test_root_ca_cert.pem"`